notanorm/sqlite.py

```diff
@@ -218,7 +221,7 @@ def __info_to_model(cls, info):
 
     def model(self):
         """Get sqlite db model: dict of tables, each a dict of rows, each with type, unique, autoinc, primary"""
-        res = self.query("SELECT name, type from sqlite_master")
+        res = self.query("SELECT name, type from sqlite_master", no_capture=True)
```

Comment:
I'm not sure how I feel about this. Getting the model is far from free, and we do it implicitly in a lot of places.

In many cases, I'd think you absolutely would want to capture that this happened, especially if it's somewhere perf sensitive.